### ISSUE TITLE
Missing fonts?...

### ISSUE THREAD

**Issue by GwynethLlewelyn** [ChristosBouronikos/typora-nord-theme#2]
Hello there,

I'm a super-fan of Nord, too, and was thinking of doing exactly what you've already done three months ago 🤣 — but when viewing your screenshots, I admit that I didn't like the fonts you've chosen. This is really easy to change, so I went into your CSS and saw what had been your choices.

The choice of 'Open Sans' is a peaceful one (most people will have it pre-installed), but I noticed that for those who haven't got it installed, you added `./onedark/700i.woff`. Well, I guess that directory is missing from your distribution! I'm assuming that you've copied the CSS from https://theme.typora.io/theme/OneDark/ and forgot to change the references... (you can get [Open Sans directly from Google](https://fonts.google.com/specimen/Open+Sans) but there are lots of alternatives out there).

Note that [Arctic Ice Studio](https://www.nordtheme.com/), the Nord theme creators, uses the free ['Rubik'](https://fonts.google.com/specimen/Rubik) font for _most_ of their website, but sometimes uses ['Inter'](https://fonts.google.com/specimen/Inter) as well, especially for smaller text. Again, this is all a matter of personal preferences...

As for 'Consolas', this font is included only on people having Windows 10 or Microsoft 365 (formerly known as [Office 365](https://www.microsoft.com/en-gb/microsoft-365/office-365)). I'm a Mac user, but I'm also a Microsoft 365 subscriber, so I _do_ have that font installed; but those who don't use either Windows 10 or Microsoft 365 (think: Linux users and all Mac users without a Microsoft 365 subscription...). 'Consolas' is a rather beautiful font, but you cannot distribute it, so I'd recommend falling back to 'Source Code Pro' or 'Noto Sans Mono' or 'Droid Sans Mono' (all of these are free to distribute). I'm personally a fan of ['Fira Code'](https://github.com/tonsky/FiraCode) but I admit that the overall style is different from 'Consolas'!

FYI, for monotype fonts, Arctic Ice Studio uses ['Source Code Pro'](https://fonts.google.com/specimen/Source+Code+Pro) on their website (weight 600).

Anyway, thanks for saving me a lot of work! 😆

**Comment by ChristosBouronikos** [comment]
Heyyy, I honestly love the Nord color palette so much that I was looking for a good fork for typora but couldn't find any, so I had to make one for myself.

The thing is I didn't know any CSS during the summer, so most of the things you see is trial and error until something works from the simplest GPL3 licensed theme I could find.

I'm already taking an advanced website technologies course this semester (and I'm actually acing it 😊) while doing my Master's at the same time and working full time.

My schedule is hectic and I'm pretty swamped with paper deadlines during this time, considering the situation of my job and the long hours in front of the computer it's unfeasible for me to update the theme at this time.

I plan to remake it from the ground up during the holidays now that I have the knowledge to do so.

I'm so sorry for any inconveniences I caused, and I'm so glad there are more people that needed this in their lives. 

I also found out about a serif font called [BioRhyme](https://fonts.google.com/specimen/BioRhyme) on Google fonts and it looks really nice. Check it out, since you like nice fonts! 

Any other mistakes or criticism is very much welcome.
